File: 240927/배수의 개수/number-of-multipliers.js
const fs = require("fs");
const number = fs.readFileSync(0).toString().trim().split("\n");

let cnt_3 = 0;
let cnt_5 = 0;

for (let i = 0; i <= 9; i++) {
    if (number[i] % 3 === 0) {
        cnt_3++;
    }
    if (number[i] % 5  === 0 ) {
        cnt_5++;
    }
}

console.log(cnt_3, cnt_5)

File: 251027/배수의 개수/number-of-multipliers.py
# 리스트 컴프리헨션 : [표현식 for 변수 in range(n) if]

nums = [int(input()) for _ in range(10)]
cnt3 = 0
cnt5 = 0

for i in range(10):
    if nums[i] % 3 == 0:
        cnt3 += 1
    if nums[i] % 5 == 0:
        cnt5 += 1
    else:
        continue

print(cnt3, cnt5)